target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -154,6 +154,48 @@ define Device/tplink_archer-d50-v1
 endef
 TARGET_DEVICES += tplink_archer-d50-v1
 
+define Device/tplink_archer-d7-v1
+  ATH_SOC := qca9558
+  DEVICE_TITLE := TP-Link Archer D7 v1
```

Comment:
Please update to use DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT as introduced in:
https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5